fastlane/spec/actions_specs/import_from_git_spec.rb

```diff
@@ -24,7 +24,11 @@
           source_directory_path = Dir.mktmpdir("fl_spec_import_from_git_source")
 
           Dir.chdir(source_directory_path) do
-            `git init -b master`
+            if Gem::Version.new(`git --version`.strip.split(' ').last) >= Gem::Version.new("2.28.0")
```

Comment:
`git --version` outputs `git version 2.32.0 (Apple Git-132)` on my machine, so I think that this check would be incorrect.

Isn't there any other way to retrieve only the version, without needing to parse it?

Reply:
[Apparently not](https://git-scm.com/docs/git-version) :(

Reply:
Maybe using https://regex101.com/r/fImCOu/1 would be a better idea?

Reply:
@revolter yes, better. I even made most groups optional to parse a potential "3" version.

Reply:
I didn't do that so as not to match the `132` in my example, but I guess that `.match()` only returns the first match, so it's all good 👍🏻